turbo.json

```diff
@@ -36,7 +36,7 @@
       "outputs": ["src/gen"]
     },
     "test": {
-      "dependsOn": ["build"]
+      "dependsOn": ["generate-model", "build"]
```

Comment:
Why this change? Test already depends on build, and build already depends on generate-model